docs/features/feature-guides/properties/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Structured Properties
+<FeatureAvailability/>
+
+**DataHub Structured Properties** provide a structured and validated approach to attaching metadata to your Data Assets, enabling organizations to enforce consistency and quality in metadata collection.
```

Comment:
We can simplify this sentence. The main thing to highlight here is "custom attributes". Users will lock onto the intuition once they hear this - many other systems have similar features (usually named "custom properties") 

**Structured Properties** allow you to add custom, validated properties to any entity type in DataHub. Using structured properties, you can enable data discovery & governance based on attributes unique to your organization.